modules/imaging_browser/php/viewsession.class.inc

```diff
@@ -703,7 +703,9 @@ class ViewSession extends \NDB_Form
         $candidate            =& \Candidate::singleton($timePoint->getCandID());
         $subjectData['pscid'] = $candidate->getPSCID();
         $subjectData['dob']   = $candidate->getCandidateDoB();
-        $subjectData['edc']   = $candidate->getCandidateEDC();
+        if ($tpl_data['useEDC'] === 'true') {
```

Comment:
$config = \NDB_Config::singleton();
        if ($config->getSetting('useEDC') === 'true') {
            $subjectData['edc'] = $candidate->getCandidateEDC();
        }

Reply:
@kongtiaowang Thank you for reviewing!!

I reverted the changes to `viewsesison.class.inc` back to what it was and made the changes in the `Candidate.class.inc` instead as suggested by @johnsaigle in issue #4184 .